app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -288,26 +293,24 @@ class PipelineSampleReport extends React.Component {
     );
   }
 
-  applySearchFilter(
-    searchTaxonId,
-    includedCategories,
-    input_taxons,
-    includedSubcategories = []
-  ) {
+  applySearchFilter(searchTaxonId = this.state.search_taxon_id) {
```

Comment:
Why cant we move searchTaxonId below like the remaining arguments?

Reply:
you're right, we totally can. in a subsequent diff i think i will rename some of these functions to be more accurately descriptive of what they're doing (since applySearchFilter also applies category filters)